Editor/Editors/CinemachineStoryboardEditor.cs

```diff
@@ -11,7 +11,7 @@ namespace Cinemachine.Editor
     [InitializeOnLoad]
     internal class CinemachineStoryboardMute
     {
-        const string StoryboardGlobalMuteMenuName = "Cinemachine/Storyboard Global Mute";
+        const string StoryboardGlobalMuteMenuName = "GameObject/Cinemachine/Storyboard Global Mute";
```

Comment:
Leave this in the Cinemachine menu for now (maybe as the only thing).
We need to create a preferences page to move it to.  We can save that for another PR.
(so just revert this change)